### PR TITLE
Update Backup setting wording

### LINKED ISSUE #1732: Game specific backup - settings text
The text on the settings page related to the game specific toggles needs updating to something more user friendly. 

Also can an experimental tag be added to those options please? 

@captainsandypants Any suggestions for the text needed? 

Please move the game specific settings under the experimental section.

### PR DESCRIPTION
**Pull request by Al12rs**
- fixes #1732

### CODE REVIEW

File: tests/Games/NexusMods.Games.RedEngine.Tests/Cyberpunk2077SynchronizerTests.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public async Task ContentIsIgnoredWhenSettingIsSet()
     {
         // Get the settings
         var settings = ServiceProvider.GetRequiredService<ISettingsManager>().Get<Cyberpunk2077Settings>();
-        settings.IgnoreContentFolder = true;
+        settings.DoFullGameBackup = true;
```
Comment (erri120) [inline tests/Games/NexusMods.Games.RedEngine.Tests/Cyberpunk2077SynchronizerTests.cs:19]: Need to be inversed.

File: tests/Games/NexusMods.Games.RedEngine.Tests/Cyberpunk2077SynchronizerTests.cs [hunk 2]
```diff
@@ -42,7 +42,7 @@ public async Task ContentIsIgnoredWhenSettingIsSet()
         (await FileStore.HaveFile(notIgnoredHash)).Should().BeTrue("The file was not ignored"); 
         
         // Now disable the ignore setting
-        settings.IgnoreContentFolder = false;
+        settings.DoFullGameBackup = false;
```
Comment (erri120) [inline tests/Games/NexusMods.Games.RedEngine.Tests/Cyberpunk2077SynchronizerTests.cs:45]: Need to be inversed.

**Review by erri120 — CHANGES_REQUESTED** [review]

**Review by erri120 — APPROVED** [review]

**Comment by captainsandypants** [comment]
@Al12rs did this also change the section the settings are contained in? Looks like it's still in the new "Game specific" section

**Comment by Al12rs** [comment]
Fixed

**Pull request merged**